### PR TITLE
small start (footer & navbar)

### PR DESCRIPTION
**Pull request by AW2077**
Dodałam wszystkie img ze starego repo, ale z tego co widzę, będzie je trzeba przejrzeć i część usunąć.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,10 +1,19 @@
 <!doctype html>
-<html lang="en">
+<html lang="pl">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrom=1" />
```
Comment (szymonkadas) [inline index.html]: nitpick: chrome=1 (it is chrom)

**Review by szymonkadas — APPROVED** [review]
Overally gj, one nitpick to change before merging

**Comment by szymonkadas** [comment]
Oh and forgot to say: redundant photos are from saved and deleted folders, they're old versions, Łukasz was supposed to verify which files to delete etc. but ig he fully agreed upon these changes, we can maintain those in saved folder and delete those in deleted.

**Comment by AW2077** [comment]
już miałam klikać haha

**Comment by szymonkadas** [comment]
th-4010855568

**Pull request merged**